client/utils/auth.py

```diff
@@ -36,6 +34,23 @@
 TOKEN_ENDPOINT = '/oauth/token'
 ERROR_ENDPOINT = '/oauth/errors'
 
+COPY_MESSAGE = """
+Copy the following URL and open it in a web browser. To copy, highlight the URL
+and press Ctrl+Shift+C (Windows, Linux) or Cmd+C (Mac), or right-click and
+select "Copy".
+""".strip()
```

Comment:
If this copy step turns out to be too hard & frequent enough - we can come up with a few techniques to make it easier

- A help page. If you need help go to https://okpy.org/client/faq (Where we have video examples for different browsers)
- Automatically copy this URL to the clipboard. Will J tried this in #164 - it turned out to be a little more pain than it was worth. 
- Have the ok server have a short url for that redirects to the OOB url. For example: https://okpy.org/client/login`

Given that students only have to do this once and hopefully can get through it - I think what we have now will be fine.